index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <script src="https://kit.fontawesome.com/4f02d055e5.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@200&family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Aladdin</title>
+</head>
+
+<body>
+    <div class="container"> 
+    <section class="header">
+        <nav>
+            <ul>
```

Comment:
- [ ] The padding for this `ul` is alittle off the figma design. Adjust the icon padding/margin to match the logo just like in the design.
![Screenshot from 2021-08-14 03-56-57](https://user-images.githubusercontent.com/65501535/129429798-8b676e09-a288-4086-b4a2-d9f8cd82da4d.png)

- [ ] Also, adjust the alignment of the social icons a little more to the left to match the text above it.